src/components/InactiveProjectsList.js

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from 'react'
+import { useHistory } from 'react-router-dom';
+import { useQuery } from '@apollo/client';
+import { Grid } from '@material-ui/core'
+import { orderBy } from 'lodash'
+
+import ProjectTile from './ProjectTile'
+import LoadingProgress from './LoadingProgress'
+import { GET_INACTIVE_PROJECTS, GET_INACTIVE_PROJECTS_COUNT } from '../operations/queries/ProjectQueries';
+import FormGroup from '@material-ui/core/FormGroup'
+import FormControlLabel from '@material-ui/core/FormControlLabel'
+import Switch from '@material-ui/core/Switch'
+import Box from '@material-ui/core/Box'
+
+const InactiveProjectsList = (props) => {
```

Comment:
Instead of creating a whole new component for the list of inactive projects is better if you reuse ProjectsList component but just passing a list of inactive project